zavod/zavod/helpers/names.py

```diff
@@ -1,9 +1,18 @@
-from typing import Optional
+from typing import List, Optional, cast
 from normality import collapse_spaces
 from followthemoney.util import join_text
+import re
 
+from zavod.context import Context
 from zavod.entity import Entity
 
+REGEX_AND = re.compile(r"(\band\b|&|\+)", re.I)
+REGEX_JUST_A_NAME = re.compile(r"^[a-z]+, [a-z]+$", re.I)
```

Comment:
I think `[a-z]` will mean the regex will miss any non-ascii characters. AFAIK Pythons built-in `re` doesn't support character classes, but there is a library called `regex` that will do it: 
https://pypi.org/project/regex/

Reply:
Just took a 2nd look and `\w+` and the `re.U` works as well :)

Reply:
yeah - I used a-z because \w includes numbers and I've seen companies referenced as just a number. This is really quite a specific solution to US sort of lists. I'll improve the regex name to indicate intent a bit more clearly. Of course we can do better with this case but I need to not go deeper here

Reply:
actually matching too widely is the safer thing to do and we'll notice if it becomes unbearable from warnings